pages/1.12/cli/command-reference/dcos-help/index.md

```diff
@@ -16,19 +16,92 @@ The `dcos help` command will display DC/OS CLI help information.
 ```bash
 dcos help <subcommand>
 ```
+Output of `dcos help` (without options) should look like this:
```

Comment:
This help output seems incorrect for the 1.12 (0.7.x) CLI.
It should look something like this:
```
Usage:
  dcos [command]

Commands:
  auth
      Authenticate to DC/OS cluster
  cluster
      Manage your DC/OS clusters
  config
      Manage the DC/OS configuration file
  help
      Help about any command
  job
      Deploy and manage jobs in DC/OS
  marathon
      Deploy and manage applications to DC/OS
  node
      View DC/OS node information
  package
      Install and manage DC/OS software packages
  plugin
      Manage CLI plugins
  service
      Manage DC/OS services
  task
      Manage DC/OS tasks

Options:
  --version
      Print version information
  -v, -vv
      Output verbosity (verbose or very verbose)
  -h, --help
      Show usage help

Use "dcos [command] --help" for more information about a command.
```

Reply:
Further investigation:
I spun up a 1.12 cluster and loaded the CLI according to instructions. Here is the info I got from `dcos --version`:

dcos --version
dcoscli.version=0.6.1
dcos.version=1.12.0
dcos.commit=20fa047bbd37188ccb55f61ab9590edc809030ec
dcos.bootstrap-id=51c3e1a0a5b7fdace4abccc998e970df07eb606e`

You will see that even though this is 1.12, I am still only able to download 0.6.1. Apparently this is the only version available to 1.12 and 1.13. Even the upgrade/patching instructions do not tell users how to select a specific version of the CLI.

**How did you get 0.7.1 loaded?**